idaes/tests/test_headers.py

```diff
@@ -39,11 +51,14 @@ def test_headers(package_root, patterns):
         # modify patterns to match the files that should have headers
         ff = FileFinder(package_root, glob_patterns=patterns)
         has_header, missing_header = detect_files(ff)
-        if len(missing_header) > 0:
+        # ignore empty files (probably should add option in 'detect_files' for this)
+        nonempty_missing_header = list(filter(lambda p: p.stat().st_size > 0, missing_header))
```

Comment:
(minor coding suggestion) consider using a list comprehension instead of `list` + `filter` + `lambda`:

```py
nonempty_missing_header = [p for p in missing_header if p.stat().st_size > 0]
```

Reply:
whatever?